src/conn/conn_sweep.c

```diff
@@ -292,8 +292,13 @@ __sweep_server(void *arg)
 	__wt_seconds(session, &last);
 	for (;;) {
 		/* Wait until the next event. */
-		__wt_cond_wait(session, conn->sweep_cond,
-		    min_sleep * WT_MILLION, __sweep_server_run_chk);
+		if (FLD_ISSET(conn->timing_stress_flags,
+			WT_TIMING_STRESS_AGGRESSIVE_SWEEP))
+			__wt_cond_wait(session, conn->sweep_cond,
+				min_sleep * 10000, __sweep_server_run_chk);
```

Comment:
We will check more often with reduced sleep, but we will still run lookaside sweep or file handle sweep at the same duration as before. Let us talk offline to figure out how we can resolve that.

Reply:
One way is to reduce the `conn->sweep_interval` config value [here](https://github.com/wiredtiger/wiredtiger/blob/92719d6bc9a83ce45c337db6a67adcc1354cca32/src/conn/conn_sweep.c#L387) and also reduce the `WT_LAS_SWEEP_SEC` in the aggressive mode.